### PR TITLE
5535 style fixes

### PR DESCRIPTION
**Pull request by bthaile**
list of clean style fix up tasks.

### CODE REVIEW

File: src/utils/format-date.js [hunk 1]
```diff
@@ -62,3 +69,8 @@ export function getBeginDate(periodString) {
   }
   return beginDate.unix()
 }
+
+export function dateHasPassed(unitTimestamp) {
```
Comment (stephensprinkle) [inline src/utils/format-date.js]: Was this supposed to be `unixTimestamp`?

File: src/modules/reporting/components/reporting-open/reporting-open.jsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ export default class ReportingOpen extends Component {
     toggleFavorite: PropTypes.func.isRequired,
     loadMarketsInfo: PropTypes.func.isRequired,
     isLogged: PropTypes.bool.isRequired,
+    isMobile: PropTypes.bool,
```
Comment (stephensprinkle) [inline src/modules/reporting/components/reporting-open/reporting-open.jsx]: Update this to `isRequired`

File: src/utils/format-date.js [hunk 3]
```diff
@@ -25,10 +31,11 @@ export function formatDate(d) {
     value: date,
     simpleDate: `${date.getUTCDate()} ${months[date.getUTCMonth()]}`,
     formatted: `${months[date.getUTCMonth()]} ${date.getUTCDate()}, ${date.getUTCFullYear()} ${utcTimeTwelve.join(':')} ${utcAMPM}`, // UTC time
+    formattedShort: `${shortMonths[date.getUTCMonth()]} ${date.getUTCDate()}, ${date.getUTCFullYear()} ${utcTimeTwelve.join(':')} ${utcAMPM}`, // UTC time
     formattedLocal: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} ${localTimeTwelve.join(':')} ${localAMPM} (UTC ${localOffset})`, // local time
     formattedLocalShort: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} (UTC ${localOffset})`, // local time
     full: date.toUTCString(),
-    timestamp: date.getTime()
+    timestamp: date.getTime() / 1000
```
Comment (stephensprinkle) [inline src/utils/format-date.js]: Roll this back, as most often, we’re receiving a timestamp from the backend that already has millis trimmed.

File: src/utils/format-date.js (hunk 1 above)
Comment (bthaile) [reply src/utils/format-date.js]: yea, typo will fix

**Comment by stephensprinkle** [comment]
Couple small comments; otherwise, looks good!

**Pull request merged**